edgemesh/cmd/edgemesh.go

```diff
@@ -15,7 +15,8 @@ func main() {
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 	pflag.Parse()
 
-	pkg.Register()
+	// TODO need parse edgemesh config file before Register @kadisi
+	//pkg.Register()
```

Comment:
Why comments here, doesn't it affect the edgemesh work?

Reply:
Now edgemesh is not running as a separate process. If we need it to run a separate process , we need to define a configuration file for him.But I don't think this is necessary for the moment because mesh is now running as a module of edgecore。